pkgs/top-level/stage.nix

```
@@ -64,6 +64,30 @@
 } @args:
 
 let
+  # This is a function from parsed platforms (like
+  # stdenv.hostPlatform.parsed) to parsed platforms.
+  makeStaticParsedPlatform = parsed:
```

```suggestion
  makeMuslParsedPlatform = parsed:
```
Thanks for factoring this out! I wasn't aware of the duplication.